modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -81,4 +97,199 @@ enter the specific IP address you want and related information. For example:
+
+a|`ignition.config.url`
+
+a|Identifies the location of an Ignition config that you build to pass selected values to be used by the live installer. This argument can be used for adding disk partitioning or other features you want to have in place before the actual installation runs.
+
+|===
+
+[discrete]
+== `coreos-installer` options for ISO install
+
+You can run the `coreos-installer` command to identify various artifacts to include, to work with disk partitions, and to set up networking. In some cases, you can configure features on the live system and copy them to the installed system.
+
+This allows you to prepare the permanent system in a variety of ways before first boot.
+
+The following table shows the options and subcommands you can pass to the `coreos-installer` command from a shell prompt during a live install.
+
+.`coreos-installer` command-line options, arguments, and subcommands
+[source,adoc]
+|===
+
+2+|_Command-line options_
+
+|*Option* |*Description*
+
+a| `-u`, `--image-url <url>`
+a|Specify the image URL manually.
+
+a| `-f`, `--image-file <path>`
+a|Specify a local image file manually.
+
+a|`-i,` `--ignition-file <path>`
+a|Embed an Ignition config from a file.
+
+a|`I`, `--ignition-url <URL>`
+a|Embed an Ignition config from a URL.
+
+a|`--ignition-hash <digest>`
+a|Digest `type-value` of the Ignition config.
+
+a|`-p`, `--platform <name>`
+a|Override the Ignition platform ID.
+
+a|`--append-karg <arg>...`
+a|Append the default kernel argument.
+
+a|`--delete-karg <arg>...`
+a|Delete the default kernel argument.
+
+a|`-n`, `--copy-network`
+a|Copy the network configuration from the install environment.
+
+a|`--network-dir <path>`
+a|For use with `-n`. Default is `/etc/NetworkManager/system-connections/`.
+
+a|`--save-partlabel <lx>..`
+a|Save partitions with this label glob.
+
+a|`--save-partindex <id>...`
+a|Save partitions with this number or range.
+
+a|`--offline`
+a|Force offline installation.
+
+a|`--insecure`
+a|Skip signature verification.
+
+a|`--insecure-ignition`
+a|Allow Ignition URL without HTTPS or hash.
+
+a|`--architecture <name>`
+a|Target CPU architecture. Default is `x86_64`.
+
+a|`--preserve-on-error`
+a|Do not clear partition table on error.
+
+a|`-h`, `--help`
+a|Print help information.
+
+2+|_Command-line argument_
+
+|*Argument* |*Description*
+
+a|`<device>`
+a|The destination device.
+
+2+|_coreos-installer embedded Ignition commands_
+
+|*Command* |*Description*
+
+a|`$ coreos-installer iso ignition embed <options> <ISO_image> <Ignition_config>`
+a|Embed an Ignition config in an ISO image.
+
+a|`coreos-installer iso ignition show <options> <ISO_image> <Ignition_config>`
+|Show the embedded Ignition config from an ISO image.
+
+a|`coreos-installer iso ignition remove <options> <ISO_image> <Ignition_config>`
+a|Remove the embedded Ignition config from an ISO image.
+
+2+|_coreos-installer ISO Ignition options_
```

Comment:
Not all of these are accepted by all subcommands.

Reply:
Added this as a note for this section.